src/components/section/ohri-form-section.component.tsx

```diff
@@ -111,7 +113,9 @@ const OHRIFormSection = ({ fields, onFieldChange }) => {
                         <PreviousValueReview
                           previousValue={previousFieldValue}
                           displayText={
-                            fieldDescriptor.questionOptions.rendering == 'checkbox'
+                            fieldDescriptor.questionOptions.rendering === 'date'
+                              ? formatDate(previousFieldValue)
+                              : fieldDescriptor.questionOptions.rendering == 'checkbox'
                               ? previousCheckboxDisplayValue
                               : previousFieldValue?.display
```

Comment:
This is getting overly nested and looks like something that might grow with more unique rendering types